packages/ui/src/components/sign-up/index.tsx

```diff
@@ -1 +1,15 @@
-export { SignUpComponent as SignUp } from './sign-up';
+import { Root as SignUpRoot } from '@clerk/elements/sign-up';
+
+import { SignUpContinue } from './steps/continue';
+import { SignUpStart } from './steps/start';
+import { SignUpVerifications } from './steps/verifications';
+
+export function SignUp() {
+  return (
+    <SignUpRoot>
+      <SignUpStart />
+      <SignUpVerifications />
+      <SignUpContinue />
+    </SignUpRoot>
+  );
+}
```

Comment:
Nit: my preference would be to keep this `sign-up.tsx` and re-export from `index.ts`

Reply:
Works for me, I've been cautious to introduce any barrel files though.